Sources/Sonar/Models/ModelsData.swift

```diff
@@ -28,66 +28,42 @@ extension Reproducibility {
+    public static let AlliOSAreas: [Area] = [
+        Area(1, "Accelerate Framework"), Area(2, "Accessibility"), Area(3, "Accounts Framework"),
+        Area(4, "App Store"), Area(5, "App Switcher"), Area(6, "Audio"), Area(7, "Audio Toolbox"),
+        Area(8, "AVFoundation"), Area(9, "AVKit"), Area(10, "Battery Life"), Area(11, "Bluetooth"),
+        Area(12, "Calendar"), Area(13, "CallKit"), Area(14, "CarPlay"),
+        Area(15, "Cellular Service (Calls / Data)"), Area(16, "CFNetwork Framework"), Area(17, "CloudKit"),
+        Area(18, "Contacts"), Area(19, "Contacts Framework"), Area(20, "Control Center"),
+        Area(21, "Core Animation"), Area(22, "Core Bluetooth"), Area(23, "Core Data"),
+        Area(24, "Core Foundation"), Area(25, "Core Graphics"), Area(26, "Core Image"),
+        Area(27, "Core Location"), Area(28, "Core Media"), Area(29, "Core Motion"),
+        Area(30, "Core Spotlight"), Area(31, "Core Telephony Framework"), Area(32, "Core Video"),
+        Area(33, "EventKit"), Area(34, "External Accessory Framework"), Area(35, "FaceTime"),
+        Area(36, "Foundation"), Area(37, "GameplayKit"), Area(38, "GLKit"), Area(39, "Health App"),
+        Area(40, "HealthKit"), Area(41, "Home App"), Area(42, "Home Screen"), Area(43, "HomeKit"),
+        Area(44, "iCloud"), Area(45, "Image I/O"), Area(46, "Intents Framework"), Area(47, "IOKit"),
+        Area(48, "iPod Accessory Protocol (iAP)"), Area(49, "iTunes Connect"), Area(50, "iTunes Store"),
+        Area(51, "Keyboard"), Area(52, "Local Authentication Framework"), Area(53, "Location Services"),
+        Area(54, "Lock Screen"), Area(55, "Mail"), Area(56, "MapKit"), Area(57, "Maps"), Area(58, "MDM"),
+        Area(59, "Media Player Framework"), Area(60, "Messages"), Area(61, "Messages Framework"),
+        Area(62, "Metal"), Area(63, "Model I/O"), Area(64, "Multipeer Connectivity Framework"),
+        Area(65, "Music"), Area(66, "Network Extensions Framework"), Area(67, "Notes"),
+        Area(68, "Notification Center"), Area(69, "NotificationCenter Framework"), Area(70, "Notifications"),
+        Area(71, "PassKit"), Area(72, "Phone App"), Area(73, "Photos"), Area(74, "Photos Framework"),
+        Area(75, "Profiles"), Area(76, "PushKit"), Area(77, "QuickLook Framework"), Area(78, "Reminders"),
+        Area(79, "ReplayKit"), Area(80, "Safari"), Area(81, "Safari Services"), Area(82, "SceneKit"),
+        Area(83, "Security Framework"), Area(84, "Setup Assistant"), Area(85, "Simulator"), Area(86, "Siri"),
+        Area(87, "Social Framework"), Area(88, "Software Update"), Area(89, "Spotlight"),
+        Area(90, "SpriteKit"), Area(91, "StoreKit"), Area(92, "System Slow/Unresponsive"),
+        Area(93, "SystemConfiguration Framework"), Area(94, "TouchID"), Area(95, "UIKit"),
+        Area(96, "UserNotifications Framework"), Area(97, "VPN"), Area(98, "Wallet"), Area(99, "WebKit"),
+        Area(100, "Wi-Fi"), Area(101, "Xcode"), Area(102, "Something not on this list"),
```

Comment:
Usage of this outside brisk will be weird, since we don't want people to create a random `Area` this is why I initially used fixed areas. Same as `classification`, `product`, `reproducibility`, etc.

(See: https://github.com/br1sk/Sonar/blob/master/Example/main.swift)